target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4019-eap2200.dts

```diff
@@ -0,0 +1,284 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "EnGenius EAP2200";
+	compatible = "engenius,eap2200";
+
+	chosen {
+		/*
+		 * Overwrite u-boot mtdparts cmdline parameters
+		 */
+		 bootargs-append = " ubi.mtd=ubi root=/dev/ubiblock0_1";
```

Comment:
Aren't you with the OEM/ODM? Because you could just fix this in the u-boot (i.e. not pass invalid bootargs).